tests/test_generate_data.py

```diff
@@ -65,3 +66,24 @@ def test_params_same_from_sim_data(self):
                  grouped_data['n_custs'])
         assert ((np.array(list(bbgb_params.values())) - np.array(
             bbtf._unload_params('alpha', 'beta', 'gamma', 'delta'))) < 0.1).all()
+
+
+bg_params = OrderedDict([('T', 100), ('r', 0.24), ('alpha', 4.41),  ('a', 0.79),
+                           ('b', 2.43), ('size', 800)])
+
+
+class TestTransactionalDataGeneration():
+
+
+    def test_beta_geometric_nbd_model_transactional_data(self):
+        np.random.seed(188898)
+        summary_original = beta_geometric_nbd_model(**bg_params)[['frequency', 'recency', 'T']]
+        np.random.seed(188898)
+        summary_test = summary_data_from_transaction_data(beta_geometric_nbd_model_transactional_data(**bg_params),
+        'customer_id', 'date')
+
+        summary_original['recency'] = summary_original['recency'].apply(np.ceil)  # summary_data_from_transaction_data makes regency floor
+        summary_original = summary_original.reset_index(drop=True)
+        summary_test = summary_test.reset_index(drop=True)
+        # assert
+        summary_original.equals(summary_test)
```

Comment:
`assert summary_original.equals(summary_test)`

Reply:
I've added `assert` in each test. Also I've added more tests for different type of `T` and different `freq`. Thank you!